docs/usage.md

```diff
@@ -13,6 +13,31 @@ General process:
 4. set the environment variables ([Containerfile example](#write-the-dockerfile-or-containerfile))
 5. run the build ([container build example](#build-the-container))
 
+## Configuration
```

Comment:
I'm not sure if this section best fits the `usage` doc... I think `usage` is more focused on practical usage guide, and this explanatory section feels out of place to me.

I'd argue this could fit into a section in the main README (right after basic usage), or maybe keep the small example as it currently is in the `basic usage` section and create a link to a standalone doc with the detailed explanation.